Formula/node.rb

```diff
@@ -52,8 +52,8 @@ class Node < Formula
   # We track major/minor from upstream Node releases.
   # We will accept *important* npm patch releases when necessary.
   resource "npm" do
-    url "https://registry.npmjs.org/npm/-/npm-8.9.0.tgz"
-    sha256 "19a36ab5340cbe5b17d1aa0fa18a5934bbb5b8157db055706b155d98b6f122b7"
+    url "https://registry.npmjs.org/npm/-/npm-8.12.0.tgz"
+    sha256 "4f4be1c100db1c26564a156fdae1ebe6c126061103e561732252f8a0ba451ef8"
```

Comment:
node 18.3.0 bundles npm 8.11.0 (nodejs/node#43210). The upgrade to 8.12.0 (nodejs/node#43291) has not been merged yet.

Reply:
Yeah, good call out.

Reply:
my little test
```
$ nvm use 18
Now using node v18.3.0 (npm v8.11.0)
```